<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>373</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="Main_Title">
        <property name="font">
         <font>
          <family>DejaVu Sans</family>
          <pointsize>13</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Escolher Cores</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="margin">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="FirstRing_Label">
          <property name="text">
           <string>Primeiro Anel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="FirstRing">
          <property name="currentIndex">
           <number>-1</number>
          </property>
          <property name="duplicatesEnabled">
           <bool>false</bool>
          </property>
          <property name="frame">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="SecondRing_Label">
          <property name="text">
           <string>Segundo Anel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="SecondRing"/>
        </item>
        <item>
         <widget class="QLabel" name="ThirdRing_Label">
          <property name="text">
           <string>Terceiro Anel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="ThirdRing"/>
        </item>
        <item>
         <widget class="QLabel" name="Tolerance_Label">
          <property name="text">
           <string>Tolerãncia</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="Tolerance"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="Calc_Button">
        <property name="text">
         <string>Calculate</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="Result_Label">
        <property name="font">
         <font>
          <family>DejaVu Sans</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>0.0 Ω ± 0%</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
